lib/Invoice.js

```diff
@@ -81,8 +83,17 @@ export class Invoice {
       [ 'dijbekero', this._options.proforma ],
       [ 'logoExtra', this._options.logoImage ],
       [ 'szamlaszamElotag', this._options.invoiceIdPrefix ],
-      [ 'fizetve', this._options.paid ]
-    ], indentLevel)
+      [ 'fizetve', this._options.paid ],
+    ]
+
+    if (this._options.prepaymentInvoice) {
```

Comment:
If the `prepaymentInvoice` and `finalInvoice` mutually exclude each other then
- better to use an enum, but it will be a breaking change
- throw an exception about the inconsistent state.

If we use this hidden logic then it may cause more hours debugging time for someone who set the `prepaymentInvoice` and `finalInvoice` to true and may wonder why not get the desired result.